src/DotNet/Writer/ModuleWriterBase.cs

```diff
@@ -785,6 +785,8 @@ public abstract class ModuleWriterBase : ILogger {
 			int count = chunks.Count;
 			for (int i = 0; i < count; i++) {
 				var chunk = chunks[i];
+				// TODO: We should probably align the offset and RVA here to the chunk's required alignment!
```

Comment:
The code below assumes that all of the chunks require an alignment <= min(fileAlignment, sectionAlignment). We should verify that here too and produce an error if requested alignment is too big.